.env

```diff
@@ -1 +1,2 @@
+REACT_APP_IPFS_URL="http://109.235.70.27:5001"
```

Comment:
We are currently connecting to the IPFS node on our own server. 

I have created a follow-up to this issue that should connect to IPFS running locally first: https://github.com/UniversalDot/front-end/issues/38